acceptance-tests/src/test/java/io/blueocean/ath/offline/multibranch/ParallelNavigationTest.java

```diff
@@ -89,7 +89,7 @@ public void parallelNavigationTestInput () throws IOException, GitAPIException,
         wait.until(By.cssSelector(".btn.inputStepSubmit")).click();
 
         // and clicking on the unselected node will yield us the second branch
-        wait.until(By.cssSelector(".pipeline-node")).click();
+        wait.until(By.xpath("//*[contains(@class, 'pipeline-node')][3]")).click();
```

Comment:
I'm still hoping that a non-awful solution to this sort of fragility will occur to me like some sort of Newtonian apple-falling moment.